src/domains/profile/pages/Welcome/Welcome.stories.tsx

```diff
@@ -7,11 +7,21 @@ export default {
 	title: "Profile / Pages / Welcome",
 };
 
-export const Default = () => (
-	<div className="w-full h-full">
-		<Welcome />
-	</div>
-);
+export const Default = () => {
+	const env = {
+		profiles: () => ({
+			all: async () => new Promise((resolve) => resolve([])),
```

Comment:
This method is sync, not async. https://github.com/ArkEcosystem/platform-sdk/blob/master/packages/platform-sdk-profiles/src/repositories/profile-repository.ts#L30-L32